resources/language/English/strings.po

```diff
@@ -951,3 +951,14 @@ msgctxt "#32463"
 msgid "By Artist"
 msgstr ""
 
+msgctxt "#32464"
+msgid "Player"
+msgstr ""
+
+msgctxt "#32465"
+msgid "Use skip step settings from Kodi"
+msgstr ""
+
+msgctxt "#32466"
+msgid "Automatically seek to the selected timeline position after a second"
```

Comment:
I also noticed that this doesn't fit well when displayed in settings. Perhaps we could change it to something like:

`Automatically seek selected position after a delay`